lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
```

Comment:
We should probably add `bundler` as a dependency with `>= 1.9.4` as a version requirement, or what do you think?